src/models/createNewHouseModel.js

```diff
@@ -0,0 +1,103 @@
+const mongoose = require('mongoose')
+
+const housesSchema = new mongoose.Schema({
+    owner: {
+        type: mongoose.Schema.Types.ObjectId,
+        required: true
+    },
+    address:{
```

Comment:
The address data could also use a street suffix and street number fields. It's best not to keep the data corresponding to the fields not as a part of any of the existing fields as parsing them could be difficult.